glide.yaml

```diff
@@ -65,4 +65,19 @@ import:
   version: 0fe204460da2c8fa1babcaac196e694de8f1aaa1
 
 - package: github.com/miekg/dns
-  version: db96a2b759cdef4f11a34506a42eb8d1290c598e
\ No newline at end of file
+  version: db96a2b759cdef4f11a34506a42eb8d1290c598e
+
+- package: github.com/shirou/gopsutil
+  version: 2.1.0
+  subpackages:
+  - load
+
+- package: github.com/davecgh/go-spew
+  version: 2df174808ee097f90d259e432cc04442cf60be21
+  subpackages:
+  - spew
+
+- package: golang.org/x/sys
```

Comment:
@zcahana Any ideas why the A8 Makefile doesn't use the glide update --strip-vendor option?  That option is what collapses the vendor directory.  I see that SD registry uses, but A8 does not.  Was that intentionally removed?  Should we consider adding that back in?  I can clean up the glide.yaml to remove go-spew and sys dependencies.  Do we want to add the --strip-vendor option in this commit as well or have a separate PR?